recipes/folly/all/conanfile.py

```diff
@@ -79,16 +86,35 @@ def _required_boost_components(self):
         return ["context", "filesystem", "program_options", "regex", "system", "thread"]
 
     def validate(self):
-        if tools.cross_building(self):
-            raise ConanInvalidConfiguration("Cross-compilation not yet supported. Contributions are welcome")
-            
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, self._minimum_cpp_standard)
+        min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
+        if not min_version:
+            self.output.warn("{} recipe lacks information about the {} compiler support.".format(self.name, self.settings.compiler))
+        else:
+            if tools.Version(self.settings.compiler.version) < min_version:
+                raise ConanInvalidConfiguration("{} requires C++{} support. The current compiler {} {} does not support it.".format(
+                    self.name, self._minimum_cpp_standard, self.settings.compiler, self.settings.compiler.version))
+
+        if self.version < "2022.01.31.00" and self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("Non-Linux platforms are supported starting with version 2022.01.31.00")
```

Comment:
Is it really true?

Reply:
This means for Conan.  I'm not sure about upstream, but [Conan only has Linux packages for the current two versions](https://conan.io/center/folly).  I'm guessing they tried other platforms when those were created and they didn't work?  I'm not sure.

It could be rephrased to be mean Conan-specifically.

Reply:
Yes please.

Reply:
How about
```
Conan support for non-Linux platforms starts with version 2022.01.31.00
```